src/api/artifact/controller.go

```diff
@@ -56,6 +56,8 @@ type Controller interface {
 	Delete(ctx context.Context, id int64) (err error)
 	// Tags returns the tags according to the query, specify the properties returned with option
 	Tags(ctx context.Context, query *q.Query, option *TagOption) (total int64, tags []*Tag, err error)
+	// CreateTag creates a tag
+	CreateTag(ctx context.Context, tag *Tag) (id int64, err error)
```

Comment:
to use "attach" and "detach" instead will be easy to understand?

Reply:
The `ArtifactID` is one property of the `Tag` object, when creating a new tag, it is attached to the artifact automatically. So `Create` and `Delete` is more reasonable